pageserver/src/tenant.rs

```diff
@@ -818,7 +821,7 @@ impl Tenant {
         // We're ready for business.
         // Change to active state under the hood spawns background loops
         // The loops will shut themselves down when they notice that the tenant is inactive.
-        self.activate(true);
+        self.activate(enable_background_jobs);
```

Comment:
Why not just always start the background jobs? They won't have any work to do if there are no timelines, but that seems fine

Reply:
Because `test_pageserver_with_empty_tenants` asserts that the background jobs are not started.
I want to get rid of the entire thing alltogether, but we'll do that later.
See https://neondb.slack.com/archives/C036U0GRMRB/p1668604459676979